src/Package/Impl/project.15.0.json

```diff
@@ -13,6 +13,7 @@
     "Microsoft.VisualStudio.TextManager.Interop.12.0": "12.0.30110",
     "Microsoft.VSSDK.BuildTools": "15.0.25201-Dev15Preview2",
     "System.IO.Compression.ZipFile": "4.0.0",
+    "System.Collections.Immutable": "1.1.33-beta",
```

Comment:
Good catch, it's a higher version there. I think it doesn't matter during build, because it'll just fetch the higher one because it requires it; but I'll adjust it here as well, and check on VS15.